examples/ota-requestor-app/linux/main.cpp

```diff
@@ -117,6 +120,33 @@ OptionSet cmdLineOptions = {
 
 OptionSet * allOptions[] = { &cmdLineOptions, nullptr };
 
+// Network commissioning
+namespace {
+constexpr EndpointId kNetworkCommissioningEndpointMain      = 0;
+constexpr EndpointId kNetworkCommissioningEndpointSecondary = 0xFFFE;
```

Comment:
Is endpoing 0xFFFE actually turned on in the ZAP config for this app, with the network commissioning cluster defined on it?

Reply:
I was just about to leave a comment that I realized I didn't add the secondary endpoint.